basic-nginx/Jenkinsfile

```diff
@@ -55,50 +55,17 @@ pipeline {
       }
     }
 
-    stage('Verify Deployment to Dev') {
-      steps {
-        verifyDeployment(projectName: env.DEV, targetApp: env.APP_NAME)
-      }
-    }
-
-    stage('Promotion gate (Stage)') {
```

Comment:
@kenwilli ; instead of removing, can you add a flag that enables/disables these inputs? just so both options (as-is and how you'd like it) can be provided?

Reply:
Added a flag var, left it at false by default (which will require user to confirm proceeding) but can be set using -e so the command for basic-nginx would look like: ansible-playbook -i ./.applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e skip_manual_promotion=True